target/linux/ath79/mikrotik/config-default

```diff
@@ -28,11 +31,15 @@ CONFIG_MTD_UBI_BLOCK=y
 CONFIG_MTD_UBI_WL_THRESHOLD=4096
 CONFIG_MTD_UBI_BEB_LIMIT=20
 CONFIG_NET_DSA=y
+CONFIG_NET_DSA_QCA8K=y
+CONFIG_NET_DSA_TAG_QCA=y
```

Comment:
I would prefer if we first activate CONFIG_NET_DSA_QCA8K for the compete target and not only for some boards.

Reply:
commit c2140e32ce32b9cc60f7d408e20bdf45dce6a634 added new device with old switch driver. so also mikrotik subtarget now uses old switch driver.